testsuite/features/build_validation/init_clients/proxy.feature

```diff
@@ -39,6 +39,10 @@ Feature: Setup containerized proxy
   Scenario: Wait until the proxy host appears
     When I wait until onboarding is completed for "proxy"
 
+  Scenario: Upgrade mgrpxy tool
+    Then I upgrade "proxy" with the last "mgrpxy" version
+    And I reboot the "proxy" minion through the web UI
```

Comment:
I'm not sure if we need to wait until the reboot event is completed to avoid issue on next scenarios.